src/main/java/com/kuzmins/service/UserService.java

```diff
@@ -0,0 +1,64 @@
+package com.kuzmins.service;
+
+import com.kuzmins.model.tickets.Ticket;
+import com.kuzmins.model.tickets.TicketType;
+import com.kuzmins.model.users.User;
+import com.kuzmins.repositories.TicketRepository;
+import com.kuzmins.repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+@Transactional(readOnly = true)
```

Comment:
try to avoid to use this annotation on class level.
If any method in the class does perform write operations, you should override the `@Transactional` annotation on that method explicitly with `@Transactional(readOnly = false)`. Otherwise, changes may not be committed due to the read-only transaction settings.

Reply:
removed

Reply:
I mean remove not only `(readOnly = false)`, but also the whole `@Transactional` from class level. Try to use it on method level and only for methods, where you need to work within 1 transaction. Usually it is for write-operations or when you need to make few separate calls to repositories.

Reply:
ok, I got and fixed